package.json

```diff
@@ -43,11 +43,13 @@
     "version": "yarn clean && yarn build"
   },
   "dependencies": {
+    "@juggle/resize-observer": "^3.2.0",
     "formidable-oss-badges": "^0.3.5",
     "markdown-toc": "^1.2.0",
     "prism-react-renderer": "^1.1.1",
     "react-copy-to-clipboard": "^5.0.2",
     "react-live": "^2.2.2",
+    "react-scroll": "^1.8.1",
```

Comment:
I think `react-markdown` should be a dependency

Reply:
ah yes! I'm surprised this didn't yell at me